server/core/src/main/scala/sttp/tapir/server/metrics/Metric.scala

```diff
@@ -64,3 +64,22 @@ object MetricLabels {
     )
   )
 }
+
+object OTELMetricLabels {
```

Comment:
instead of creating a new object, let's create a new `val OpenTelemetryAttributes: MetricLabels = ...` in the `MetricsLabel` companion object. Simply another set of labels that can be used.